Specs/Number/Arabic/PercentModel.json

```diff
@@ -1,180 +1,222 @@
 [
   {
-    "Input": "100%",
+    "Input": "هي الوحيدة الذي حصلت على ١٠٠٪.",
+    "NotSupportedByDesign": "java, javascript, python",
+    "NotSupported": "dotnet",
     "Results": [
       {
-        "Text": "100%",
+        "Text": "١٠٠٪",
         "TypeName": "percentage",
         "Resolution": {
           "value": "100%"
         },
-        "Start": 0,
-        "End": 3
+        "Start": 25,
+        "End": 28
       }
     ]
   },
   {
-    "Input": " 100% ",
```

Comment:
We have planned a different PR for colloquial usage, negative cases and few variations. We will have a discussion on this before including these cases into the test specs.

Reply:
Adding cases in separate PRs is fine. But please don't delete cases that already exist in the specs unless they are invalid.
If certain cases have not yet been validated, add a "Comment" attribute stating that.
Otherwise it's very hard to track why different cases were editted/added/removed.

Reply:
Added the existing test cases with comments Pending Validation in both Number Model and Percent Model.